eZ/Publish/Core/Persistence/Legacy/Content/FieldHandler.php

```diff
@@ -332,7 +332,7 @@ public function updateFields(Content $content, UpdateStruct $updateStruct, Type
                 if (isset($updateFieldMap[$fieldDefinition->id][$languageCode])) {
                     $field = clone $updateFieldMap[$fieldDefinition->id][$languageCode];
                     $field->versionNo = $content->versionInfo->versionNo;
-                    if (isset($field->id) && array_key_exists($field->languageCode, $existingLanguageCodes)) {
+                    if (null !== $field->id && array_key_exists($field->languageCode, $existingLanguageCodes)) {
```

Comment:
What's the difference? `isset` isn't working with this magic call (I assume)?

Reply:
It is just to satisfy PHPStans complain about "issetability" of `$field->id`. Probably baseline would be sufficient solution.

Reply:
I'll need to see the error myself. `isset` should be properly respected in terms of data type change by PHPStan.

Reply:
>>> What's the difference? `isset` isn't working with this magic call (I assume)?

>> It is just to satisfy PHPStans complain about "issetability" of `$field->id`. Probably baseline would be sufficient solution.

> I'll need to see the error myself. `isset` should be properly respected in terms of data type change by PHPStan.

AFAICS, this is a DTO with public properties. What's the exact PHPStan issue @Nattfarinn ?